### ISSUE TITLE
[Errors] constructor parameter named 'static'

### ISSUE THREAD

**Issue by mhegazy** [microsoft/TypeScript#71]
``` TypeScript
'use strict'
// static as constructor parameter name should give error if 'use strict'
class test {
   constructor(static ) { }
   //!!! Identifier expected.
}
```
